Source/EventFlow/Aggregates/IAggregateRoot.cs

```diff
@@ -35,8 +35,13 @@ public interface IAggregateRoot
         IEnumerable<IAggregateEvent> UncommittedEvents { get; }
         bool IsNew { get; }
 
-        Task<IReadOnlyCollection<IDomainEvent>> CommitAsync(IEventStore eventStore, CancellationToken cancellationToken);
+        Task<IReadOnlyCollection<IDomainEvent>> CommitAsync(
+            IEventStore eventStore,
+            ISourceId sourceId,
```

Comment:
@rasmus This is a breaking change. Consider extending the IAggregateRoot interface with the old implementation. If the aggregate does not care about the sourceId then it can be omitted.